packages/yoastseo/spec/fullTextTests/testTexts/el/greekPaper.js

```diff
@@ -19,8 +19,10 @@ const paper = new Paper( content, {
 const expectedResults = {
 	introductionKeyword: {
 		isApplicable: true,
-		score: 9,
-		resultText: "<a href='https://yoa.st/33e' target='_blank'>Keyphrase in introduction</a>: Well done!",
+		score: 6,
+		resultText: "<a href='https://yoa.st/33e' target='_blank'>Keyphrase in introduction</a>: Your keyphrase or its " +
+			"synonyms appear in the first paragraph of the copy, but not within one sentence. <a href='https://yoa.st/33f' " +
+			"target='_blank'>Fix that</a>!",
 	},
```

Comment:
The keyphrase Ελληνική γλώσσα appears in the first paragraph, but not within one sentence:

Η φωνολογία, η μορφολογία, η σύνταξη και το λεξιλόγιο της **γλώσσας** δείχνουν τόσο συντηρητικά όσο και καινοτόμα στοιχεία σε ολόκληρη την ιστορική πορεία της **γλώσσας** από την αρχαία έως τη σύγχρονη περίοδο. Η διαίρεση σε συμβατικές περιόδους είναι σχετικά αυθαίρετη, ειδικά επειδή σε όλες τις περιόδους ύπαρξης της η αρχαία **ελληνική** έχει απολαύσει υψηλό κύρος και οι εγγράμματοι άνθρωποι χρησιμοποιούσαν πολλά δάνεια από τα αρχαία **ελληνικά**.